src/NzbDrone.Core/Indexers/Definitions/Rarbg/RarbgParser.cs

```diff
@@ -41,12 +44,17 @@ public IList<ReleaseInfo> ParseResponse(IndexerResponse indexerResponse)
 
             if (jsonResponse.Resource.error_code.HasValue)
             {
-                if (jsonResponse.Resource.error_code == 20 || jsonResponse.Resource.error_code == 8
-                    || jsonResponse.Resource.error_code == 9 || jsonResponse.Resource.error_code == 10
-                    || jsonResponse.Resource.error_code == 5 || jsonResponse.Resource.error_code == 13
-                    || jsonResponse.Resource.error_code == 14)
+                if (jsonResponse.Resource.error_code is 20 or 8 or 9 or 10 or 5 or 13 or 14)
                 {
-                    // No results, rate limit, or imdbid/tvdb not found
+                    if (jsonResponse.Resource.rate_limit is 1)
+                    {
+                        _logger.Debug("No results due to rate limiting.");
+                    }
+                    else
+                    {
+                        _logger.Debug("No results or imdbid/tvdb not found: error code {0}.", jsonResponse.Resource.error_code);
+                    }
+
```

Comment:
Would it be better to log out the text error they provide as well `jsonResponse.Resource.error`